modules/claims_api/lib/bgs_service/vnp_proc_service_v2.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class VnpProcServiceV2 < ClaimsApi::LocalBGS
+    PROC_TYPE_CD = 'POAAUTHZ'
+
+    def vnp_proc_create
+      body = Nokogiri::XML::DocumentFragment.parse <<~EOXML
+        <arg0>
+          <VnpProcDTO>
+            <vnpProcTypeCd>#{PROC_TYPE_CD}</vnpProcTypeCd>
+          </VnpProcDTO>
+        </arg0>
+      EOXML
+
+      make_request(endpoint: 'VnpProcWebServiceBeanV2/VnpProcServiceV2', action: 'vnpProcCreate', body:, key: 'return')
```

Comment:
Is this endpoint sensitive? I would recommend putting it in param store

Reply:
this exists within [bgs-ext](https://github.com/department-of-veterans-affairs/bgs-ext) and also lives within [existing cassettes in vets-api](https://github.com/department-of-veterans-affairs/vets-api/blob/de101d0b0a1a0d3a42b1ecf92d4cf8993c8abb4d/spec/support/vcr_cassettes/bgs/form674/submit.yml#L5). We'll be building a new service for multiple endpoints using this same concept. We _can_ move them to Param Store, but I don't think we'd have concerns with adding this.